src/linker/cli.py

```diff
@@ -129,6 +130,7 @@ def run_slurm_job(
     implementation_name: str,
     container_full_stem: str,
     input_data: Tuple[str],
+    config: Tuple[str],
```

Comment:
We want to pass these strings directly, not in a config file?

Reply:
We could do that. I just didn't want to deal with parsing a config file outside of the original Config object if I could help it. Remember that this command isn't really intended to be called by users - linker calls it if appropriate and I've found it useful for developing. It's actually a "hidden" function in that if you do `linker --help` you won't see this as an avaible tool.